registry_resolver/Cargo.toml

```diff
@@ -8,14 +8,18 @@ edition = "2021"
 [dependencies]
 prost = "0.11.0"
 prost-types = "0.11.0"
-tonic = "0.8.0"
+tonic = { version = "0.8.2", features = ["tls", "tls-roots", "tls-webpki-roots"] }
```

Comment:
Why are these new imports needed?

Reply:
They are needed to enable grpc over https